pkg/models/protocoldriver.go

```diff
@@ -12,19 +12,20 @@
 package models
 
 import (
-	"github.com/edgexfoundry/go-mod-core-contracts/v3/clients/logger"
 	"github.com/edgexfoundry/go-mod-core-contracts/v3/models"
+
+	"github.com/edgexfoundry/device-sdk-go/v3/pkg/interfaces"
 )
 
 // ProtocolDriver is a low-level device-specific interface used by
-// by other components of an EdgeX Device Service to interact with
+// other components of an EdgeX Device Service to interact with
 // a specific class of devices.
 type ProtocolDriver interface {
 	// Initialize performs protocol-specific initialization for the device service.
 	// The given *AsyncValues channel can be used to push asynchronous events and
 	// readings to Core Data. The given []DiscoveredDevice channel is used to send
 	// discovered devices that will be filtered and added to Core Metadata asynchronously.
-	Initialize(lc logger.LoggingClient, asyncCh chan<- *AsyncValues, deviceCh chan<- []DiscoveredDevice) error
+	Initialize(sdk interfaces.DeviceServiceSDK, asyncCh chan<- *AsyncValues, deviceCh chan<- []DiscoveredDevice) error
```

Comment:
> I feel that it would make it easier to expand functionality in the future without causing breaking changes.

This is the clever sloution, updated !